### PR TITLE
Command line

### PR DESCRIPTION
**Pull request by Hodobox**
Edit, Help, scenebuilder todo

### CODE REVIEW

File: src/main/java/graphgui/GraphPane.java [hunk 1]
```diff
@@ -8,12 +8,16 @@
 import java.util.HashMap;
 import javafx.event.ActionEvent;
 import javafx.geometry.Insets;
+import javafx.geometry.NodeOrientation;
```
Comment (kubik369) [inline src/main/java/graphgui/GraphPane.java]: Why do you have unused imports here?

File: src/main/java/graphgui/GraphPane.java [hunk 2]
```diff
@@ -346,25 +346,20 @@ public void executeCommand(String[] tokens) {
 
         File file = new File(filename);
```
Comment (kubik369) [inline src/main/java/graphgui/GraphPane.java:359]: Doesn't this throw exceptions?

File: src/main/java/graphgui/GraphPane.java [hunk 3]
```diff
@@ -418,106 +400,167 @@ public void executeCommand(String[] tokens) {
...
+
+              if (chosenEdge == null) {
+                throw new IllegalArgumentException(
+                  String.format("Hrana %d %d neexistuje", from, to)
+                );
+              }
+
+            } catch (NumberFormatException e) {
+              return  "Indexy pre edit edge maju nespravny format - ocakavaju sa dve cele cisla";
+
```
Comment (kubik369) [inline src/main/java/graphgui/GraphPane.java]: Delete this empty line

File: src/main/java/graphics/Controller.java [hunk 4]
```diff
@@ -67,6 +69,11 @@
   @FXML
   public TextArea taConsole;
 
+  public static final ArrayList<String> VERTEXCOLORS
```
Comment (kubik369) [inline src/main/java/graphics/Controller.java]: Please put an underscore into the name of the variable

File: src/main/java/graphics/Controller.java [hunk 5]
```diff
@@ -159,6 +162,60 @@ public void saveGraphHandler(ActionEvent actionEvent) {
...
+   */
+  @SuppressWarnings("checkstyle:LineLength")
+  public void displayHelpAlert() {
+    Alert alert = new Alert(Alert.AlertType.INFORMATION);
+    alert.setTitle("Supported Commands");
+    alert.setHeaderText(null);
+    alert.setResizable(true);
+    alert.getDialogPane().setMinWidth(750);
+
+    alert.setContentText(
```
Comment (kubik369) [inline src/main/java/graphics/Controller.java]: Could this be done so that the text is highlightable and the command names are in bold?

File: src/main/java/graphics/layout.fxml [hunk 6]
```diff
@@ -13,21 +13,23 @@
 <?import javafx.scene.layout.BorderPane?>
 <?import graphgui.GraphPane?>
 
+
```
Comment (kubik369) [inline src/main/java/graphics/layout.fxml]: Why did you add these empty lines?

**Review by kubik369 — CHANGES_REQUESTED** [review]

File: src/main/java/graphgui/GraphPane.java (hunk 1 above)
Comment (Hodobox) [reply src/main/java/graphgui/GraphPane.java]: no idea, had to be some sort of misclick.

**Review by kubik369 — APPROVED** [review]
